.github/workflows/InvalidationFlagger.yml

```
@@ -1,5 +1,9 @@
 name: Invalidations
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

```suggestion
on: [push]
```

The checks run seems only to work on a push event right now.

> https://docs.github.com/en/rest/reference/checks#check-runs
> Note: The Checks API only looks for pushes in the repository where the check suite or check run were created. Pushes to a branch in a forked repository are not detected and return an empty pull_requests array.